### PR TITLE
Container implantation

### PR DESCRIPTION
**Pull request by rosplk**
New container implantation detection, investigation story

### CODE REVIEW

**Comment by d1vious** [comment]
@rosplk looks like there was an error in the manifest validation https://app.circleci.com/jobs/github/splunk/security-content/4538/parallel-runs/0/steps/0-106 

```
processing manifest ./detections/container_implant_aws_detection.yml
ERROR: "'data_source' is a required property" at:
	./detections/container_implant_aws_detection.yml
	Affected Object: {"data_models": ["Vulnerabilities"], "data_sourcetypes": ["aws:cloudtrail"], "providing_technologies": ["AWS"]}
```

File: detections/container_implant_gcp_detection.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  data_models:
+    - Vulnerabilities
+  data_source:
+    - google:gcp:pubsub:messages
+  providing_technologies:
+    - GCP
+description: This search show information on uploaded containers including source user, account, action, bucket name event name, http user agent, message and destination path.
+detect:
+  splunk:
+      search: '| datamodel Cloud_Infrastructure Storage search | search Storage.event_name=storage.objects.create | table Storage.src_user Storage.account Storage.action Storage.bucket_name Storage.event_name Storage.http_user_agent Storage.msg Storage.object_path'
```
Comment (patel-bhavin) [inline detections/container_implant_gcp_detection.yml]: @rosplk : any particular reason we are not using tstats in this search?

File: detections/container_implant_aws_detection.yml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  data_sourcetypes:
+    - aws:cloudtrail
+  providing_technologies:
+    - AWS
+description: This searches show information on uploaded containers including source user, image id, source IP
+  user type, http user agent, region, first time, last time of operation (PutImage). These searches
+  are based on Cloud Infrastructure Data Model.
+detect:
+  splunk:
+      search: '| tstats count min(_time) as firstTime max(_time) as lastTime FROM datamodel=Cloud_Infrastructure.Compute where Compute.user_type!="AssumeRole" AND Compute.http_user_agent="AWS Internal" AND Compute.event_name="PutImage" by Compute.image_id Compute.src_user Compute.src Compute.region Compute.msg Compute.user_type | `drop_dm_object_name("Compute")`'
```
Comment (patel-bhavin) [inline detections/container_implant_aws_detection.yml]: @rosplk : we should add post filter macros in both the search and create respective macro ymls

File: detections/container_implant_gcp_detection.yml [hunk 3]
```diff
@@ -0,0 +1,44 @@
+asset_type: GCP GCR Container
+confidence: medium
+creation_date: '2020-02-20'
+data_metadata:
+  data_models:
+    - Vulnerabilities
```
Comment (patel-bhavin) [inline detections/container_implant_gcp_detection.yml]: incorrect datamodel

**Pull request merged**